docs/_source/tutorials-examples/population-synthesis/10_binaries_pop_syn.ipynb

```diff
@@ -229,12 +244,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "\n",
     "There are some additional metadata properties available, such as:\n",
-    "- `metallicities`: the metallicity in absolute metallicity\n",
-    "- `solar_metallicities`: the metallicities in the file in solar metallicity\n",
-    "- `number_of_systems`: the total number of systems in the Population file\n",
-    "- `indices`: the indices of the binaries in the file\n",
-    "- `columns`: the columns available in the `history` and `oneline` dataframes"
+    "\n",
+    "- `metallicities` the metallicity in absolute metallicity\n",
+    "- `solar_metallicities` the metallicities in the file in solar metallicity\n",
+    "- `number_of_systems` the total number of systems in the Population file\n",
```

Comment:
Why do we state the `number_of_systems` here again? It is already part of the previous list.

Reply:
It's a different thing to access them. The first list is what's in the `mass_per_metallicity`, the second list is class variables

Reply:
But are those different properties, as the introducing sentence `There are some additional metadata properties available, such as:` suggests?